sources/packages/forms/src/form-definitions/educationprogram.json

```diff
@@ -1697,7 +1697,7 @@
           "value": ""
         }
       ],
-      "content": "<strong>Warning! This program is ineligible for 20+ hours of study per week</strong><br/>\nStudents need to have 20 or less than hours of study per week.",
+      "content": "<strong>Warning! This program is ineligible for less than 20 hours of study per week</strong><br/>\nStudents need to have 20 or more hours of study per week.",

```

Comment:
I believe that there is a period (".") missing at the end of the second sentence (of study per week).